app/views/users/invitations/edit.html.erb

```diff
@@ -21,3 +21,15 @@
 
   <%= f.submit t("devise.invitations.edit.submit_button"), class: 'buttonGhostM smallMarginTop' %>
 <% end %>
+<div class="center">
+  <div class="smallMarginTop">
+    <div class="smallMarginBottom">
```

Comment:
I don't think we need the nested div with smallMarginBottom, wouldn't the sign up with Google button be the only button?